### PR TITLE
Deprecated chars eh

### PR DESCRIPTION
**Pull request by ehinman**
Function added to autoclean that converts deprecated characteristic names to most current versions in TADA.CharacteristicName. Updated example data affected by retired parameter names.

### CODE REVIEW

File: R/Utilities.R [hunk 1]
```diff
@@ -157,6 +158,10 @@ TADA_AutoClean <- function(.data) {
   # .data$TADA.ResultDepthHeightMeasure.MeasureUnitCode[.data$ResultDepthHeightMeasure.MeasureUnitCode == 'meters'] <- 'm'
   .data$TADA.ResultMeasure.MeasureUnitCode[.data$TADA.ResultMeasure.MeasureUnitCode == 'meters'] <- 'm'
   
+  # Substitute updated characteristic name for deprecated names
+  print("TADA_Autoclean: updating deprecated characterisitc names.")
```
Comment (cristinamullin) [inline R/Utilities.R:162]: typo

Comment (ehinman) [reply R/Utilities.R:162]: Bah! Thank you. Will fix.

File: R/Utilities.R [hunk 2]
```diff
@@ -546,6 +551,45 @@ TADA_OrderCols <- function(.data){
   
 }
 
+#' Substitute Preferred Characteristic Name for Deprecated Names
+#' 
+#' This utility function uses the WQX Characteristic domain table to substitute
+#' deprecated characteristic names with the new name in the
```
Comment (cristinamullin) [inline R/Utilities.R:557]: deprecated (i.e., retired and/or invalid)

File: R/Utilities.R [hunk 3]
```diff
@@ -546,6 +551,45 @@ TADA_OrderCols <- function(.data){
...
+#' TADA.CharacteristicName column. Used in TADA_Autoclean function on data
+#' pulled from the WQP.
```
Comment (cristinamullin) [inline R/Utilities.R:559]: TADA_SubstituteDeprecatedChars is run within TADA_Autoclean, which runs within TADA_DataRetreival and TADA_BigDataRetrieval. Therefore, deprecated characteristic names are harmonized to their current name automatically upon data retrieval. TADA_SubstituteDeprecatedChars can also be used by itself on a user supplied dataset that is in the WQX format, if desired.

**Pull request merged**